### PR TITLE
ci: fix unstable t/chaos/killchaos

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What this PR does / why we need it:

add delay before access route

also fix format of tmate actions to make timeout works

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by Yiyiyimu** [comment]
The reason might relate to https://github.com/apache/apisix-dashboard/pull/976 that we need to wait sometime to make etcd sync finished.

File: t/chaos/killetcd/killetcd.go [hunk 1]
```diff
@@ -96,8 +96,23 @@ var _ = ginkgo.Describe("Test Get Success When Etcd Got Killed", func() {
 	ginkgo.It("check if everything works", func() {
 		utils.SetRoute(e, httpexpect.Status2xx)
 		utils.GetRouteList(e, http.StatusOK)
-		time.Sleep(1 * time.Second)
-		utils.GetRoute(e, http.StatusOK)
+		var resp *httpexpect.Response
+
+		resp = utils.GetRouteIgnoreError(e)
+		// wait 1s seems not enough, wait some more time to make sure nothing goes wrong
+		if resp.Raw().StatusCode != http.StatusOK {
```
Comment (tokers) [inline t/chaos/killetcd/killetcd.go:103]: Try to use `k8s.io/apimachinery/pkg/utils/wait`: https://pkg.go.dev/k8s.io/apimachinery@v0.21.3/pkg/util/wait

**Review by spacewander — APPROVED** [review]

File: t/chaos/killetcd/killetcd.go (hunk 1 above)
Comment (Yiyiyimu) [reply t/chaos/killetcd/killetcd.go:103]: Actually there are already [checks](https://github.com/apache/apisix/pull/4654/files/39a5af086a7a48d752f7cd8a052459945d8a2fa4#diff-aef74626794d502616587a665ada7300d710401a1ceff2c687e9f30cb4975b6bR66) on apisix deployment when start apisix. But the problem - **new configuration needs seconds to be synced** - still exists.

The possible reason I guess is related to https://github.com/apache/apisix-dashboard/pull/976 and it seems we need longer waiting time for this situation in kubernetes. Have you met with this problem before?

Comment (tokers) [reply t/chaos/killetcd/killetcd.go:103]: Nope.

**Review by tokers — APPROVED** [review]

**Pull request merged**